### PR TITLE
feat: Add SDK Actions missing in UI Schema to Adaptive Flow Edge Menu

### PR DESCRIPTION
**Pull request by tdurnford**
## Description
feat: Add SDK Actions missing in UI Schema to Adaptive Flow Edge Menu

## Task Item
#minor

## Screenshots
image

### CODE REVIEW

File: Composer/packages/extension-client/src/hooks/useMenuConfig.ts [hunk 1]
```diff
@@ -29,6 +33,21 @@ export function useMenuConfig(): { menuSchema: MenuUISchema; forceDisabledAction
...
+    Object.entries(sdkDefinitions).forEach(([$kind, action]) => {
+      if (
+        $kind.startsWith('Microsoft') &&
+        $kind !== SDKKinds.AdaptiveDialog &&
+        !implementedMenuSchema[$kind] &&
+        actionImplementsDialog(action)
+      ) {
+        implementedMenuSchema[$kind] = {
+          label: getFallbackLabel($kind),
+          submenu: ['Other'],
```
Comment (tdurnford) [inline Composer/packages/extension-client/src/hooks/useMenuConfig.ts:46]: Should we localize this? If so, we need to add the format message package.

Comment (a-b-r-o-w-n) [reply Composer/packages/extension-client/src/hooks/useMenuConfig.ts:46]: @yeze322 would know more here but I think the menu is currently not localized or localized elsewhere.

**Comment by cwhitten** [comment]
@srinaath @tdurnford let's remember to bring this up in the authoring standup tomorrow for discussion on naming/grouping of these

**Comment by srinaath** [comment]
@cwhitten that sounds good. Will get it in the standup tomorrow.

**Comment by cwhitten** [comment]
@srinaath was this discussed in the authoring standup? Was there an outcome?

**Comment by garypretty** [comment]
For any actions that we do think should be added to Composer today, that have an obvious home, we should add them to the right menu if possible.  A couple of questions....

- Is there someone pushing for these (or a subset of these) actions to be added in R13?  I would like to understand if there is a pressing need here, or if we potentially have time and do this in R14.
- Can anybody tell me how 'Cancel all dialogs' differs from 'Cancel all active dialogs' (I would suggest Cancel all Dialogs should go on the Dialog menu, but having both of these seems confusing).
- If we are going to add GoToAction then I think the dialog menu probably makes sense for that too.

In summary, I would like it if we can avoid adding another menu to the list if possible, as it is already pretty long.  If we really do need these in R13 though, then I am happy with the 'Other' option if we don't have a suitable other place for things to live.

**Comment by cwhitten** [comment]
@garypretty the following issues from @tomlm spawned this exercise.

https://github.com/microsoft/BotFramework-Composer/issues/7035
https://github.com/microsoft/BotFramework-Composer/issues/7034

The expectation I believe is that if the component ships with the SDK, it needs to also ship with Composer. I believe this is fine for most of these showing up, but some need custom UX in order to be usable and to show them without it would confuse folks. 

RE: component behavior--'m not sure on this. It may require a discussion with @Stevenic and @tomlm

**Comment by tdurnford** [comment]
@garypretty There are two actions in the SDK Schema - `Microsoft.CancelAllDialogs` and `Microsoft.CancelDialog` - that have the same title and description. I'm honestly not sure how they're different and why the same title. @tomlm Can you provide an explanation? 

https://github.com/microsoft/BotFramework-Composer/blob/c0640064cdb9393edecc7c62203b104c72bea69c/Composer/packages/server/schemas/sdk.schema#L1047
https://github.com/microsoft/BotFramework-Composer/blob/c0640064cdb9393edecc7c62203b104c72bea69c/Composer/packages/server/schemas/sdk.schema#L1114
